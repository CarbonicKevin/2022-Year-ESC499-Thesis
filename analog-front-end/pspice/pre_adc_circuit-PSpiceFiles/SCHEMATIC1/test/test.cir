** Profile: "SCHEMATIC1-test"  [ C:\Users\kevin\OneDrive\University\Y4S1_2024_Fall_Semester\Thesis\pre_adc_circuit\pspice\pre_adc_circuit-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\cds_spb_home\cdssetup\OrCAD_PSpiceTIPSpice_Install\17.4.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 10ns 0 
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
