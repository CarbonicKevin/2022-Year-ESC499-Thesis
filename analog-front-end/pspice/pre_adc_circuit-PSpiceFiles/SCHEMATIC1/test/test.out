
**** 08/10/22 17:33:55 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\kevin\OneDrive\University\Y4S1_2024_Fall_Semester\Thesis\pre_adc_circuit\pspice\pre_adc_ci


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\cds_spb_home\cdssetup\OrCAD_PSpiceTIPSpice_Install\17.4.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 10ns 0 
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source PRE_ADC_CIRCUIT
.EXTERNAL OUTPUT Vout
R_delyiannis_filter_R2         delyiannis_filter_N03304 VOUT  16 TC=0,0 
C_delyiannis_filter_C1         delyiannis_filter_N03272
+  delyiannis_filter_N03304  1p  TC=0,0 
R_delyiannis_filter_R3         N05463 delyiannis_filter_N03272  6366 TC=0,0 
C_delyiannis_filter_C2         delyiannis_filter_N03272 VOUT  1p  TC=0,0 
R_delyiannis_filter_R1         N05098 delyiannis_filter_N03272  6366 TC=0,0 
X_delyiannis_filter_U1         N05463 delyiannis_filter_N03304 VOUT N04411
+  N04453 OPA855
X_unity_amplifier_U1         N05098 N07232 N07232 N04411 N04453 OPA855
V_Vsrc+         N04411 0 5
V_Vsrc-         0 N04453 5
V_V_offset         N05463 0 0.5
V_V1         N05098 0  AC 1
+SIN 0 1 500Meg 0 0 0

**** RESUMING test.cir ****
.END

**** 08/10/22 17:33:55 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\kevin\OneDrive\University\Y4S1_2024_Fall_Semester\Thesis\pre_adc_circuit\pspice\pre_adc_ci


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_delyiannis_filter_U1.X_E_N.DVN 
          IS  100.000000E-18 
          KF  316.227800E-12 


               X_delyiannis_filter_U1.X_I_NP1.DNVF 
          IS  100.000000E-18 
          KF  316.227800E-12 


               X_delyiannis_filter_U1.X_U1.DNVF 
          IS  100.000000E-18 
          KF  316.227800E-12 


               X_unity_amplifier_U1.X_E_N.DVN 
          IS  100.000000E-18 
          KF  316.227800E-12 


               X_unity_amplifier_U1.X_I_NP1.DNVF 
          IS  100.000000E-18 
          KF  316.227800E-12 


               X_unity_amplifier_U1.X_U1.DNVF 
          IS  100.000000E-18 
          KF  316.227800E-12 


**** 08/10/22 17:33:55 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\kevin\OneDrive\University\Y4S1_2024_Fall_Semester\Thesis\pre_adc_circuit\pspice\pre_adc_ci


 ****     Resistor MODEL PARAMETERS


******************************************************************************




               X_delyiannis_filter_U1.R_NOISELESS 
  T_Measured   27            
   T_Current -273.15         
           R    1            


               X_unity_amplifier_U1.R_NOISELESS 
  T_Measured   27            
   T_Current -273.15         
           R    1            


**** 08/10/22 17:33:55 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\kevin\OneDrive\University\Y4S1_2024_Fall_Semester\Thesis\pre_adc_circuit\pspice\pre_adc_ci


 ****     Voltage Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_delyiannis_filter_U1.X_ESD_OUT.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .5          
        VOFF     .45         


               X_delyiannis_filter_U1.X_S1._S1 
         RON    1.000000E-03 
        ROFF   20.000000E+03 
         VON     .04         
        VOFF    -.05         


               X_delyiannis_filter_U1.X_S2._S2 
         RON    1.000000E-03 
        ROFF   20.000000E+03 
         VON     .04         
        VOFF    -.05         


               X_unity_amplifier_U1.X_ESD_OUT.ESD_SW 
         RON   50            
        ROFF    1.000000E+12 
         VON     .5          
        VOFF     .45         


               X_unity_amplifier_U1.X_S1._S1 
         RON    1.000000E-03 
        ROFF   20.000000E+03 
         VON     .04         
        VOFF    -.05         


               X_unity_amplifier_U1.X_S2._S2 
         RON    1.000000E-03 
        ROFF   20.000000E+03 
         VON     .04         
        VOFF    -.05         


ERROR(ORPSIM-16583): Detected an imported model containing transistors or diodes. For such models, PSpice for TI supports a minimum of one and maximum of three traces. Reduce the number of traces and simulate again.


ABORTING SIMULATION
**** 08/10/22 17:33:55 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ C:\Users\kevin\OneDrive\University\Y4S1_2024_Fall_Semester\Thesis\pre_adc_circuit\pspice\pre_adc_ci


 ****     JOB STATISTICS SUMMARY


******************************************************************************



Node counts:
  Top level (NUNODS)                =           8
  External (NCNODS)                 =         171
  Total (NUMNOD)                    =         171

Total device count (NUMEL)          =         387
  Capacitors (C)                    =          52
  Diodes (D)                        =          12
  VCVS (E)                          =          22
  VCCS (G)                          =          58
  CCVS (H)                          =           2
  Current Sources (I)               =          18
  Inductors (L)                     =           4
  Resistors (R)                     =         195
  VSwitches (S)                     =           8
  Voltage Sources (V)               =          12

Number of subcircuits (X)           =          34

Matrix statistics:
  Matrix size (NSTOP)               =         211
  Initial no. elements (NTTAR)      =         848
  No. elements w/ fillin (NTTBR)    =         848
  No. fillins (IFILL)               =           0
  No. overflows (NTTOV)             =           0
  No. LU operations (IOPS)          =           0
  Percent sparsity (PERSPA)         =      98.095

Analysis statistics:
  No. total time points (NUMTTP)    =           0
  No. rejected time points (NUMRTP) =           0
  No. iterations (NUMNIT)           =           0

Load Threads                        =           1

Runtime statistics:                       Seconds      Iterations
  Matrix load                       =        0.00
  Matrix solution                   =        0.00               1
  Readin                            =        1.31
  General setup                     =        0.00
  CMI setup                     =        0.00
  Setup                             =        0.00
  DC sweep                          =        0.00               0
  Bias point                        =        0.00               0
  AC and noise                      =        0.00               0
  Total transient analysis          =        0.00
  Output                            =        0.00
  Overhead                          =         .05
  License check-out time            =        4.09
  Total job time (using Solver 1)   =        1.31
